.github/workflows/update-sync-param-files.yaml

```diff
@@ -0,0 +1,35 @@
+name: update-sync-param-files
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  sync-param-files:
```

Comment:
[style]

```suggestion
  update-sync-param-files:
```

Reply:
Please check this commit
https://github.com/autowarefoundation/autoware_launch/pull/75/commits/bd49be9fdc1248a9bd4c775e5e8769e9af44ea1c